site/class-1-bar/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
You’re adding stuff to `site/class-1-bar/` instead of `site/class-1-bar/larsmarjew` here too!

Maybe start over? And ping me for help?